packages/logging-controller/src/LoggingController.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  BaseController,
```

Comment:
Be advised that there are two versions of BaseController in this repo, v1 and v2. If you just see a controller import `BaseController`, then that's v1, but we want to use v2 for all new controllers going forward. You can see an example of a controller that uses v2 here: https://github.com/MetaMask/core/blob/main/packages/ens-controller/src/EnsController.ts

Reply:
Refactored this whole file to use BaseControllerV2 and a slightly different overall architecture.